common/lib/xmodule/xmodule/modulestore/split_mongo/mongo_connection.py

```diff
@@ -328,6 +328,8 @@ def get_structure(self, key, course_context=None):
 
                 with TIMER.timer("get_structure.find_one", course_context) as tagger_find_one:
                     doc = self.structures.find_one({'_id': key})
+                    if doc is None:
```

Comment:
@adampalay I'm currently digging into this code to try to figure that out. The error message in the ticket points to `doc` being `None` but I want to be able to test this change to make sure it will actually silence the error DevOps is seeing.